osu.Game.Rulesets.Osu/Objects/Drawables/DrawableSlider.cs

```diff
@@ -106,12 +108,6 @@ public DrawableSlider(Slider s)
         private int currentRepeat;
         public bool Tracking;
 
-        public override double FadeInDuration

```

Comment:
I highly suspect that fade in of the slider itself will be desynced from fade in of the initial circle once it's changed after creation. Which mods will do.